### PR TITLE
fixed code block text background color

### PR DESCRIPTION
**Pull request by dogusata**
## Problem
With the new version of VSCode, they've implemented a global styling for code blocks to give background color to the texts inside them which also effects the code blocks in the Q Chat too. 
Screenshot 2024-01-02 at 14 30 31

## Solution
[MynahUI 3.3.2](https://github.com/aws/mynah-ui/releases/tag/v3.3.2) adds code block background color as inherited from parent. 
Screenshot 2024-01-02 at 14 30 38

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ppiliaev — APPROVED** [review]

**Review by hayemaxi — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -5548,7 +5673,6 @@
             "version": "1.1.0",
             "resolved": "https://registry.npmjs.org/@istanbuljs/load-nyc-config/-/load-nyc-config-1.1.0.tgz",
             "integrity": "sha512-VjeHSlIzpv/NyD3N0YuHfXOPDIixcA1q2ZV98wsMqcYlPmv2n3Yb2lYP9XMElnaFVXg5A7YLTeLu6V84uQDjmQ==",
-            "dev": true,
```
Comment (justinmk3) [inline package-lock.json:5551]: Why was this changed for so many dependencies?

**Comment by justinmk3** [comment]
Thanks for the clear explanation. This seems like it has customer impact, please review https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog and add a changelog item.
